samples/zigbee/ncp/README.rst

```diff
@@ -80,14 +80,14 @@ The ``UART0`` pins are configured by Devicetree :file:`overlay` files for each s
 Communication through USB
 -------------------------
 
-You can change the communication channel from the default UART to nRF USB by using the :file:`prj_usb.conf` configuration file, adding the ``-DCONF_FILE='prj_usb.conf'`` flag to your build.
+You can change the communication channel from the default UART to nRF USB by using the :file:`overlay-via-usb.conf` overlay configuration file, adding the ``-DOVERLAY_CONFIG='overlay-via-usb.conf'`` flag to your build.
 
 See :ref:`cmake_options` for instructions on how to add this flag to your build.
 For example, when building on the command line, you can do so as follows:
```

Comment:
What is the procedure when using SES? It is still the default platform and we need it covered.